activesupport/test/time_zone_test.rb

```diff
@@ -365,11 +365,19 @@ def test_iso8601_with_ambiguous_time
     assert_equal Time.utc(2014, 10, 25, 22, 0, 0), zone.parse("2014-10-26T01:00:00")
   end
 
-  def test_iso8601_with_invalid_value_parseable_by_date__iso8601
+  def test_iso8601_with_ordinal_date_value
+    zone = ActiveSupport::TimeZone["Eastern Time (US & Canada)"]
+
+    twz = zone.iso8601("21087")
+    assert_equal [0, 0, 0, 28, 03, 2021], twz.to_a[0, 6]
```

Comment:
Can you change this to match the assertion style in other tests, e.g.

```
twz = zone.iso8601("21087")
assert_equal Time.utc(2021, 3, 28, 0, 0, 0), twz.time
assert_equal zone, twz.zone 
```

That way we ensure nothing funny happens to fractional seconds

Reply:
@pixeltrix thanks for the review - amended.